problems/removeEvenStrings.js

```diff
@@ -6,8 +6,9 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+    let arr = 'hello';
```

Comment:
arr will be given to us as an argument by the person calling the function, so we don't want to redefine it to a "random" string.